recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "2020.1" %}
-{% set buildnum = "217" %}  [linux]
-{% set buildnum = "216" %}  [not linux]
+{% set buildnum = "217" %}  # [linux]
+{% set buildnum = "216" %}  # [not linux]
```

Comment:
Ahhhh I cannot read. Selectors have to be `# [stuff]` otherwise they are not proper yaml comments.